mods/ra/maps/ant-01/rules.yaml

```diff
@@ -0,0 +1,95 @@
+FireAnt:
+	Inherits: Ant
+	Tooltip:
+		Name: Giant Fire Ant
+		GenericName: Fire Ant
+	Mobile:
+		Speed: 80
+	Armament:
+		Weapon: Flamer
+	Health:
+		HP: 70000
+ScoutAnt:
+	Inherits: Ant
+	Tooltip:
+		Name: Giant Scout Ant
+		GenericName: Scout Ant
+	Mobile:
+		Speed: 99
+	Health:
+		HP: 50000
+	AutoTarget:
+		ScanRadius: 7
+Ant:
+	Tooltip:
+		GenericName: Warrior Ant
+	Mobile:
+		Speed: 50
+	Health:
+		HP: 80000
+MEDI:
+	Valued:
+		Cost: 500
+SPY:
+	Buildable:
+		Prerequisites: ~disabled
+SPY.England:
+	Buildable:
+		Prerequisites: ~disabled
+MECH:
+	Buildable:
+		Prerequisites: ~disabled
+MRJ:
+	Buildable:
+		Prerequisites: ~disabled
+APC:
+	Buildable:
+		Prerequisites: ~disabled
+STNK:
+	Buildable:
+		Prerequisites: ~disabled
+TRUK:
+	Buildable:
+		Prerequisites: ~disabled
+SPY:
+	Buildable:
+		Prerequisites: ~disabled
+SNIPER:
+	Buildable:
+		Prerequisites: ~disabled
```

Comment:
Snipers are disabled by default. Removing the custom override should have been enough to take it out.